Exercise-1/Expenses.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Expenses</title>
+    </head>
+    <body>
+        <table>
+            <tr>
```

Comment:
Try using tags to split content from headers. That is useful when writing semantic html.